packages/wrangler/tsup.config.ts

```diff
@@ -38,6 +38,7 @@ function embedWorkersPlugin({
 						sourcemap: process.env.SOURCEMAPS !== "false",
 						sourcesContent: false,
 						metafile: true,
+						external: ["cloudflare:email", "cloudflare:workers"],
```

Comment:
NIT: this can take regular expressions. So maybe?

```suggestion
						external: /^cloudflare:/,
```